src/store/actionCreators/user.ts

```diff
@@ -205,7 +201,7 @@ export const fetchUserFromJwt = (validDecodedJwt: DecodedJwt): ThunkActionUser =
     dispatch(receiveUser(profile));
   } catch (error) {
     console.error(error);
-    dispatch(revertAcceptedTermsThenLogoutCleanly());
+    dispatch(cleanlyLogout());
```

Comment:
enforcing to remove accepted terms from the front-end lead to errors in some cases (e.g unauthorized call to persona). Therefore, I rather not enforce it too much now since we will take care of this with keycloak in the near future.